odra-casper/shared/Cargo.toml

```diff
@@ -13,4 +13,5 @@ repository.workspace = true
 [dependencies]
 casper-event-standard = { workspace = true }
 odra-casper-types = { path = "../types" }
-hex = { version = "0.4.3", default-features = false }
+odra-utils = { path = "../../utils" }
+casper-contract = { version = "3.0.0", default-features = false, features = ["std", "test-support"] }
```

Comment:
Looks like this can be removed. For the future, it better to keep `shared`s and `util`s -like crates without `casper-contract`.

Reply:
A debugging leftover, just wanted to print something out.